docs/_menu/index.html

```diff
@@ -12,6 +12,10 @@
     items_attributes: 'data-documentation="/learn/documentation/version/"'
   - menu_title: Releases
     items:
+      - menu_title: 1.3.1
```

Comment:
Thanks for adding these. Do you need one for 1.4.0 as well?

Reply:
The `releases/1.4.0` page will get added onto the `master` branch.